presto-server-rpm/pom.xml

```diff
@@ -169,12 +205,96 @@
                                 <rule>
                                     <destination>/usr/lib/presto/lib</destination>
                                 </rule>
-
                             </rules>
                         </package>
                     </packages>
                 </configuration>
             </plugin>
         </plugins>
     </build>
+
+    <profiles>
+        <profile>
+            <id>ci</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>io.fabric8</groupId>
```

Comment:
This is neat, but I would use `presto-testing-docker` because running `rpm -i` is a part of a test (not setup). However `presto-testing-docker` might be not well prepared to run and validate commands in the container. So change requested.

Additionally with `presto-testing-docker`:
 - we could avoid hardcoding port
 - it would be much easier to run test from the IDE.

Reply:
I thought about that as well, but after I already had this working. We can investigate that in the future. The port number only hard-coded inside the container. The host mapping is a random port.